build-packages/magento-scripts/lib/config/php-storm.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+
+module.exports = () => {
+    const phpStormConfiguration = {
+        xdebug: {
+            v2Port: '9111',
+            v3Port: '9003',
+            debugServerAddress: 'http://localhost',
+            serverName: 'create-magento-app',
+            runManagerName: 'create-magento-app',
+            sessionId: 'PHPSTORM',
+            path: path.join(process.cwd(), '.idea', 'workspace.xml'),
```

Comment:
If you want to join the path and include CWD to the start you can use `path.resolve`.
It will add it automatically to the start.

P.S. I have some places with this issue IMO in source code, yeah... Maybe I'm completely wrong and we need to use `path.join(process.cwd()` everywhere...